tmt/steps/provision/testcloud.py

```diff
@@ -381,6 +386,10 @@ class TestcloudGuestData(tmt.steps.provision.GuestSshData):
         default=None,
         internal=True,
     )
+    logs_directory: Optional[str] = field(
```

Comment:
We already have one, see `Guest.logdir`.

Reply:
This one would not work because of selinux, see the other comment for more details.

Reply:
Removed.